main/main.cpp

```diff
@@ -1993,6 +1995,22 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 			window_vsync_mode = DisplayServer::VSyncMode::VSYNC_DISABLED;
 		}
 	}
+	{
+		frame_buffering_count = uint8_t(GLOBAL_DEF_BASIC("display/window/vsync/buffer_count", 2u));
+		swapchain_count = uint8_t(GLOBAL_DEF_BASIC("display/window/vsync/swapchain_count", 3u));
```

Comment:
Since these can only be changed at startup, they should use ``GLOBAL_DEF_RST_BASIC`` which will prompt the user to restart if they adjust the setting. 

Personally, I think these should be ``GLOBAL_DEF_RST`` as they are settings for advanced users only. Removing BASIC means that they setting will only appear if the "advanced" toggle is toggled on.
```suggestion
		frame_buffering_count = uint8_t(GLOBAL_DEF_BASIC("display/window/vsync/buffer_count", 2u));
		swapchain_count = uint8_t(GLOBAL_DEF_BASIC("display/window/vsync/swapchain_count", 3u));
```